src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/Ink/GestureRecognizer/NativeRecognizer.cs

```diff
@@ -409,14 +408,21 @@ private int AddStrokes(MS.Win32.Recognizer.ContextSafeHandle recContext, StrokeC
                     GetPacketData(stroke, out packetDescription, out countOfBytes, out packets, out xForm);
                     if (packets == IntPtr.Zero)
                     {
-                        return -2147483640; //E_FAIL - 0x80000008.  We never raise this in an exception
+                        // This isn't actually E_FAIL, it's the Macintosh version of it from the headers. Keeping for compat.
+                        return (HRESULT)(-2147483640); // E_FAIL - 0x80000008.  We never raise this in an exception
```

Comment:
Interesting 👀 If this is used in other places we should consider creating some sort of const

Reply:
It isn't. I can see how they made the mistake though. (Another argument for source generated interop.) I thought about changing it, but for now I commented and left it as is.